portfolio/src/main/webapp/script.js

```diff
@@ -11,7 +11,6 @@
 // WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
 // See the License for the specific language governing permissions and
 // limitations under the License.
-
```

Comment:
Did you mean to delete this? Seems you'd want separation between the license header and your method documentation.